### PR TITLE
Implementation of reversing a String.

### PR DESCRIPTION
**Pull request by Ephyrall**
Uploaded a new file with a solution to the task of reversing the string.

### CODE REVIEW

File: initializereverse.java [hunk 1]
```diff
@@ -0,0 +1,34 @@
+package forSaberMentoring;
+import java.util.Scanner;
+
+public class initializereverse {
```
Comment (saber0008) [inline initializereverse.java:4]: please pay closer attention to class names
https://google.github.io/styleguide/javaguide.html#s5.2.2-class-names

File: initializereverse.java [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+      char temporary;
+      for (int i = startPoint; i < endPoint; i++) {
+         if (!isNumber(charArray[i]) && !isNumber(charArray[endPoint])) {
+            temporary = charArray[i];
+            charArray[i] = charArray[endPoint];
+            charArray[endPoint] = temporary;
+         }
+         endPoint--;
+      }
+//////////
```
Comment (saber0008) [inline initializereverse.java:24]: please remove this unnecessary comment

File: initializereverse.java [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+      System.out.println("Реверснутая строка: ");
+      for (int i = 0; i < charArray.length; i++) {
+         System.out.print(charArray[i]);
+      }
+
+   }
+   private static boolean isNumber(char number) {
```
Comment (saber0008) [inline initializereverse.java:31]: please use Character.isDigit()

File: initializereverse.java [hunk 4]
```diff
@@ -0,0 +1,34 @@
+package forSaberMentoring;
+import java.util.Scanner;
+
+public class initializereverse {
+
+   public static void main(String[] args) {
```
Comment (saber0008) [inline initializereverse.java:6]: this method has at least three responsibilities: reading user input, reversing string, priting result.
IMO its worth to decompose this code a bit.
I'd suggest you to extract the logic of reversing a string into different class.
https://www.baeldung.com/java-single-responsibility-principle

File: initializereverse.java [hunk 5]
```diff
@@ -0,0 +1,34 @@
...
+            temporary = charArray[i];
+            charArray[i] = charArray[endPoint];
+            charArray[endPoint] = temporary;
+         }
+         endPoint--;
+      }
+//////////
+      System.out.println("Реверснутая строка: ");
+      for (int i = 0; i < charArray.length; i++) {
+         System.out.print(charArray[i]);
```
Comment (saber0008) [inline initializereverse.java:27]: please don't print each character, just print full result once

**Review by saber0008 — CHANGES_REQUESTED** [review]
please organize your project using standart structure
```
- src
    - main
      - java
        - <package>
```
https://jenkov.com/tutorials/maven/directory-structure.html

**Review by Ephyrall — COMMENTED** [review]
Some review. Testing.

**Review by saber0008 — CHANGES_REQUESTED** [review]
Looks good overall but project directory structure is not entirely correct, please see comment from previous review.
Also please don't commit `.idea` folder, it is your local IDE configs.
We should only commit the source code (with some rare exceptions). All local configurations, IDE settings, etc. we should exclude using .gitignore

File: stringReverse.iml [hunk 6]
```diff
@@ -0,0 +1,11 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
[the remark this answers is not in the document]
Comment (saber0008) [reply stringReverse.iml:1]: you need to remove this config file from repository

File: src/Main.java [hunk 7]
```diff
@@ -0,0 +1,15 @@
+public class Main {
```
[the remark this answers is not in the document]
Comment (saber0008) [reply src/Main.java:1]: still missing

File: src/Main.java [hunk 8]
```diff
@@ -0,0 +1,13 @@
...
+    public static void main(String[] args) {
+        InputHandler userInput = new InputHandler();
```
Comment (saber0008) [inline src/Main.java:3]: don't forget to update variable names after rename

File: src/Main.java [hunk 9]
```diff
@@ -0,0 +1,13 @@
...
+
+        String input = userInput.getUserInput();
+
+        StringManipulator stringManipulator = new StringManipulator();
+
+        String ManipulatedString = stringManipulator.reverseStringWithoutDigits(input);
```
Comment (saber0008) [inline src/Main.java:9]: non compliant variable name

File: src/StringManipulator.java [hunk 10]
```diff
@@ -0,0 +1,22 @@
+public class StringManipulator {
+    public String reverseStringWithoutDigits(String input) {
+
+        char[] charArray = input.toCharArray();
```
Comment (saber0008) [inline src/StringManipulator.java:4]: same comment as previously
https://github.com/Ephyrall/reverseRepo/pull/1#discussion_r1437021720

File: src/StringManipulator.java [hunk 11]
```diff
@@ -0,0 +1,22 @@
...
+        int startPoint = 0;
+        int endPoint = charArray.length - 1;
+
+        char temporaryChar;
```
Comment (saber0008) [inline src/StringManipulator.java:8]: No need to duplicate variable type in name

**Review by saber0008 — CHANGES_REQUESTED** [review]
Remove `.idea` folder from repository.
Also please don't forget to update directory structure.

**Comment by Ephyrall** [comment]
> Remove `.idea` folder from repository. Also please don't forget to update directory structure.

Done

File: src/EphyralCorp/InputHandler.java [hunk 12]
```diff
@@ -0,0 +1,12 @@
+package EphyralCorp;
```
Comment (saber0008) [inline src/EphyralCorp/InputHandler.java:1]: please use only lowercase in package names 
https://docs.oracle.com/javase/tutorial/java/package/namingpkgs.html#:~:text=Naming%20Conventions,a%20programmer%20at%20example.com%20.

**Review by saber0008 — CHANGES_REQUESTED** [review]

**Review by saber0008 — APPROVED** [review]

**Pull request merged**